README.md

```diff
@@ -43,6 +51,129 @@ class MyParameterizedTest {
 }
 ```
 
+### Android tests
+
+#### Setting up the dependency
+```gradle
+repositories {
+    maven { url 'https://dl.bintray.com/qameta/maven' }
+}
+
+dependencies {
+    androidTestImplementation "io.qameta.allure:allure-kotlin-model:$LATEST_VERSION"
+    androidTestImplementation "io.qameta.allure:allure-kotlin-commons:$LATEST_VERSION"
+    androidTestImplementation "io.qameta.allure:allure-kotlin-junit4:$LATEST_VERSION"
+    androidTestImplementation "io.qameta.allure:allure-kotlin-android:$LATEST_VERSION@aar"
```

Comment:
minor: adding dependency via `@aar` is an anti-pattern, please remove it from `README`